src/test/java/HomeTest.java

```diff
@@ -0,0 +1,131 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+import java.util.List;
+
+public class HomeTest extends BaseTest{
+
+  String newPlaylistName = "Sample Edited Playlist";
+
+  @Test
+    public void HoverPLayButtonAndPlaySongs() throws InterruptedException {
+
+      //Login
+    LoginPage loginPage = new LoginPage(driver);
+    HomePage homePage = new HomePage(driver);
+
+
+    loginPage.login();
+
+
+      Assert.assertTrue(hoverPlay().isDisplayed());
+
+
+    }
+
+
+    @Test
+    public void CountSongsInPlaylist() throws InterruptedException {
+
+
+    //login
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePAge = new HomePage(driver);
+
+        loginPage.login();
+
+      //Chose a playlist by name
+      choosePlaylistByName("TestPro Playlist");
+
+      //DisplayAllSongs
+      displayAllSongs();
+      //Number of songs are  equal to number of songs displayed in the info section
+      Assert.assertTrue(getPlaylistDetails().contains(String.valueOf(songCount())));
+      // above is checking if string 1 is equal to string 2
+
+  }
+//@Test
+  public void renamePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.